### DISCUSSION TITLE
Q: Airtable feature "Automation Triggers" + "Airtable Sync"

### DISCUSSION THREAD

**Discussion by sklages** [nocodb/nocodb#5503] [Q&A] [answered]
Hi,

I am a newbie with `nocodb` and I have two simple question regarding features from `Airtable` which could not be answered with the ncodb docs (probably because I was searching with wrong keywords?)..

Are these features available in `nocoDB`? Probably with different naming?
-  Automation Triggers<sup>[1]</sup>: e.g. sending emails when there have been made changes in certain tables/fields or any other condition
- Airtable Sync<sup>[2]</sup>: you can automatically sync views (and the records within them) from one base (project?) to another

Would be great if someone could point me to corresponding docs .. 

<sup>[1]</sup>https://support.airtable.com/docs/automation-triggers
<sup>[2]</sup>https://support.airtable.com/docs/can-i-link-between-different-bases

**Comment by o1lab** [comment, marked as answer]
Triggers : we've them in the form of [webhooks](https://docs.nocodb.com/developer-resources/webhooks).

Sync is not available at the moment.

**Reply by sklages** [reply]
Is "Sync" something you are planning to implement?

**Reply by o1lab** [reply]
Is the Sync one way & that can run on a cron ? Not in the immediate roadmap but we will think about it. Creating an issue from this.
